doc/source/whatsnew/v0.17.0.txt

```diff
@@ -80,6 +80,15 @@ Other enhancements
 
 - ``regex`` argument to ``DataFrame.filter`` now handles numeric column names instead of raising ``ValueError`` (:issue:`10384`).
 
+- Reindex now has a ``tolerance`` argument that allows for finer control of :ref:`basics.reindex_fill`:
+
+  .. ipython:: python
+
```

Comment:
yes that's a much better location. points to the fact that there is a section in missing.rst that sort of deals with reindexing with filling, so maybe open an issue to clarify this (e.g. put pointers back-forth or something). I think the section is missing.rst is prob older.